lib/Doctrine/Migrations/Configuration/Configuration/Exception/InvalidConfigurationFormat.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Configuration\Configuration\Exception;
+
+use Doctrine\Migrations\Configuration\Exception\ConfigurationException;
+use LogicException;
+
+final class InvalidConfigurationFormat extends LogicException implements ConfigurationException
+{
+    public static function new() : self
+    {
+        return new self('The provided configuration file can not be parsed.');
```

Comment:
Suggestion: include the filename and detected type in the exception message for easier debugging.